app/models/tp-attribute.js

```diff
@@ -32,29 +32,29 @@ const Validations = buildValidations({
       max: 64,
     }),
   ],
-  fieldName: [
+  weight: [
     validator('presence', true),
-    validator('length', {
-      max: 64,
+    validator('number', {
+      allowString: true,
+      gte: 0,
     }),
   ],
-  initial: [
+  path: [
     validator('presence', true),
     validator('length', {
       max: 64,
     }),
   ],
-  substitute: [
+  type: [
```

Comment:
API does not return type attribute, The correct name of this attr is cgType. See API doc
https://github.com/cgrates-web/cgrates_web_jsonapi/blob/master/API.md#cgrateswebjsonapitpattributecontrollershow